### PR TITLE
Added loadConfig, warnings for unexpected messages and config generator script.

### PR DESCRIPTION
**Pull request by vlad-nikityuk**

### CODE REVIEW

File: configure_nova.sh [hunk 1]
```diff
@@ -4,8 +4,8 @@ inventory=$2
 nova_compress=$3
 nova_autoconverge=$4
 nova_concurrent_migrations=$5
+nova_max_downtime=$6
 
 export ANSIBLE_HOST_KEY_CHECKING=False
 
-/usr/local/Cellar/ansible/1.9.4/bin/ansible-playbook $playbook_path/nova_flags.yml -i $inventory --extra-vars "{\"nova_compress\":$nova_compress, \"nova_autoconverge\":$nova_autoconverge, \"nova_concurrent_migrations\": $nova_concurrent_migrations}"
-
+/usr/local/bin/ansible-playbook $playbook_path/nova_flags.yml -i $inventory --extra-vars "{\"nova_compress\":$nova_compress, \"nova_autoconverge\":$nova_autoconverge, \"nova_concurrent_migrations\": $nova_concurrent_migrations, \"nova_max_downtime\":$nova_max_downtime}"
```
Comment (timofei-durakov) [inline configure_nova.sh]: please create variable for this instead, smth like 
ANSBLE=/usr/local/bin/ansible-playbook
And then update these ansible calls with $ANSIBLE ...

File: stop_telegraph.sh [hunk 2]
```diff
@@ -3,4 +3,4 @@ inventory=$2
 
 export ANSIBLE_HOST_KEY_CHECKING=False
 
-/usr/local/Cellar/ansible/1.9.4/bin/ansible-playbook $playbook_path/stop.yml -i $inventory
\ No newline at end of file
+/usr/local/bin/ansible-playbook $playbook_path/stop.yml -i $inventory
```
Comment (timofei-durakov) [inline stop_telegraph.sh]: ditto

File: start_telegraph.sh [hunk 3]
```diff
@@ -6,4 +6,4 @@ lm_scenario=$4
 
 export ANSIBLE_HOST_KEY_CHECKING=False
 
-/usr/local/Cellar/ansible/1.9.4/bin/ansible-playbook $playbook_path/start.yml -i $inventory --extra-vars "{\"lm_run\": \"$lm_run\",\"lm_scenario\":\"$lm_scenario\"}"
+/usr/local/bin/ansible-playbook $playbook_path/start.yml -i $inventory --extra-vars "{\"lm_run\": \"$lm_run\",\"lm_scenario\":\"$lm_scenario\"}"
```
Comment (timofei-durakov) [inline start_telegraph.sh]: ditto

**Comment by vlad-nikityuk** [comment]
upd

**Comment by vlad-nikityuk** [comment]
Cool

**Pull request merged**